interface/main/messages/js/reminder_appts.js

```diff
@@ -0,0 +1,524 @@
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package OpenEMR
+ * @author MedEx <support@MedExBank.com>
+ * @link http://www.open-emr.org
+ */
+
+var labels=[];
+var postcards=[];
+var show_just;
+
+$(function() {
+  $( "#datepicker1" ).datepicker({ dateFormat: "yy-mm-dd" });
+  $( "#datepicker2" ).datepicker({
+                                 dateFormat: "yy-mm-dd",
+                                 onSelect: function(dateText, inst) {
+                                    $('#'+inst.id).attr('value',dateText);
+                                 }
+                                 });
+  });
+
+
+/*
+ * Function to find a patient in the DB
+ * This pop-up is the standard openEMR file find_patient_popup.php
+ * It returns pid, lname, fname, dob to function "setpatient" below
+ * which then populates the form with the select patient data
+ */
+function recall_name_click(field) {
+    dlgopen('../../main/calendar/find_patient_popup.php?pflag=0', '_blank', 500, 400);
+}
+
+/*
+ * Function to insert patient data into addRecall fields
+ * pid is sent to server for the data to display
+ */
+function setpatient(pid, lname, fname, dob) {
+    var f = document.forms['addRecall'];
+    f.new_recall_name.value = lname + ', ' + fname;//+ '&nbsp; ('+dob+')'+''+pid;
+                                                   //go get the rest of the data
+                                                   //var id_here = document.getElementById('myCanvas_'+zone);
+                                                   //var dataURL = id_here.toDataURL();
+    top.restoreSession();
+    $.ajax({
+           type: "POST",
+           url: "save.php",
+           data: {
+           'pid'        : pid,
+           'action'     : 'new_recall'
+           }
+           }).done(function(result) {
+                   obj = JSON.parse(result);
+                   if (obj.DOLV >'') {
+                       //check to see if this is an already scheduled appt for the future
+                       //if so, do you really want a recall? Ask.
+                       //Maybe that appt needs to be removed...
+                        var now = moment(); //new Date()).format('YYYY-MM-DD'); //todays date
+                        var dolv = moment(obj.DOLV); // another date
+                        var duration = dolv.diff(now,'days');
+                        if (duration > '0') { //it's a future appt dude!
+                            alert('NOTE:  This patient has an appointment scheduled for ' + obj.DOLV +' already...');
```

Comment:
Also, should reword so the phrase is not broken up by the variable (although won't always be as eloquent, can then support translations much easier):
`NOTE: This patient already has an appointment scheduled for`